include/nil/actor/zk/snark/systems/plonk/placeholder/verifier.hpp

```diff
@@ -339,26 +317,25 @@ namespace nil {
                         F[4] = lookup_argument[1];
                         F[5] = lookup_argument[2];
                         F[6] = lookup_argument[3];
-                        F[7] = lookup_argument[4];
```

Comment:
Can we simple define `F` as F = {permutation_argument[0], permutation_argument[1], ...}

Reply:
Yes, fixed.